capstone/src/main/java/com/google/univiz/UniversityData.java

```diff
@@ -0,0 +1,67 @@
+package com.google.univiz;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class UniversityData {
+
+  public static UniversityData create(
+      String name,
+      String city,
+      float latitude,
+      float longitude,
+      int flagMainCampus,
+      int urbanizationDegree,
+      float admissionRate,
+      float avgSAT,
+      int numOfUndergrads,
+      int avgCost,
+      int studentsInDebt,
+      int carnegieSizeDegree,
+      float numOfMen,
+      float numOfWomen) {
+    return new UniversityData(
+        name,
+        city,
+        latitude,
+        longitude,
+        flagMainCampus,
+        urbanizationDegree,
+        admissionRate,
+        avgSAT,
+        numOfUndergrads,
+        avgCost,
+        studentsInDebt,
+        carnegieSizeDegree,
+        numOfMen,
+        numOfWomen);
```

Comment:
Calling the UniversityData constructor doesn't work for abstract classes. Even if it did, default constructors take no arguments.